client/blocks/comments/post-comment-list.jsx

```diff
@@ -224,6 +227,25 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	renderCommentManageLink = () => {
+		const { siteId, postId, translate } = this.props;
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
+		return (
+			<Button
+				className="comments__manage-comments-button"
+				href={ `/comments/all/${ siteId }/${ postId }` }
+				borderless
+			>
+				<Gridicon icon="chat" />
+				<span>{ translate( 'Manage', { context: 'manage comments' } ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Manage', { context: 'Login to G Suite Manage'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/43093)


:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `admin bar menu group label`
- `Login to G Suite Manage`

 Would it make sense to reuse one of the above?